Classes/Settings/DefaultReactionDetailController.swift

```diff
@@ -110,12 +104,11 @@ class DefaultReactionDetailController: UITableViewController {
     }
 
     private func updateSections() {
-        tableView.performBatchUpdates({
-            if enabledSwitch.isOn {
-                self.tableView.insertSections(IndexSet(integer: 1), with: .top)
-            } else {
+        tableView.performBatchUpdates({ [weak self] in
+            guard let `self` = self else { return }
```

Comment:
There's no retain cycle w/ `performBatchUpdates`, no need for `weak`

Also if there were, can avoid the `guard` with

```swift
self?.tableView.insertSections(...)
```

Much cleaner

(but should just remove the `weak` here)